builtin/mainmenu/tab_about.lua

```diff
@@ -172,10 +172,18 @@ return {
 			"scrollbar[15,0.1;0.4,6.9;vertical;scroll_credits;0]"
 
 		-- Render information
+		local active_renderer_info = fgettext("Active renderer:") .. " " ..
+			core.formspec_escape(core.get_active_renderer())
 		fs = fs .. "style[label_button2;border=false]" ..
-			"button[0.1,6;5.3,1;label_button2;" ..
-			fgettext("Active renderer:") .. "\n" ..
-			core.formspec_escape(core.get_active_renderer()) .. "]"
+			"button[0.1,6;5.3,0.5;label_button2;" .. active_renderer_info .. "]"..
+			"tooltip[label_button2;" .. active_renderer_info .. "]"
```

Comment:
I've added a tooltip because it's now just one line, and with #13321, the active renderer name can get very long.